source/client/fastapi_engine.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import FastAPI
+import server.database as database
+import common.mqtt_messages as mqttt_messages
+import client.storage as storage
+
+app = FastAPI()
+
+
+@app.get("/")
+async def root():
+    """
+    Root of the project.
+    """
+    return {"message": "Hello World"}
+
+@app.get("/get-messages")
+async def view_available_recipes():
+    """
+    Returns list of stored messages from the MQTT channel
+    """
+
+    return {"test": storage.test_storage}
+
```

Comment:
De ce clientul are un server de HTTP?

Clientul in principiu lucreaza de pe un device foarte limitat, care nu are capacitatea sa ruleze un server cum ar fi FastAPI, si toata comunicarea ar trebui facuta prin MQTT.